README.md

```diff
@@ -213,6 +211,26 @@ For example: `client.list({'kind': 'user', 'inspect': True})`
 | search           | Search for resources based on specified query                |
 | inspect          | List the metadata for resources                              |
 
+#### `def list_permitted_roles(list_permitted_roles_data: ListPermittedRolesData)`
+
+Lists the roles which have the named permission on a resource.
+
+#### `def list_members_of_role(data: ListMembersOfData)`
+
+Lists the roles which have the named permission on a resource.
+
+#### `def create_token(create_token_data: CreateTokenData)`
+
+Create token/s for hosts with restrictions
+
+#### `def create_host(create_host_data: CreateHostData)`
+
+Create new host using the hostfactory endpoint
```

Comment:
```suggestion
Use hostfactory token to create host
```
Actually it also can be used to rotate api key of existing host, but I'm not sure if we documented it.